src/input/source/hidraw/lego_xinput.rs

```diff
@@ -262,12 +233,16 @@ fn translate_event(event: event::Event) -> NativeEvent {
                 Capability::Mouse(Mouse::Button(MouseButton::Middle)),
                 InputValue::Bool(value.pressed),
             ),
+            event::GamepadButtonEvent::ShowDesktop(value) => NativeEvent::new(
+                Capability::Gamepad(Gamepad::Button(GamepadButton::Keyboard)),
+                InputValue::Bool(value.pressed),
+            ),
+            event::GamepadButtonEvent::AltTab(value) => NativeEvent::new(
+                Capability::Gamepad(Gamepad::Button(GamepadButton::RightPaddle2)),
+                InputValue::Bool(value.pressed),
+            ),
```

Comment:
Should we create a new button capability for this? Maybe like `GamepadButton::SwitchWindow`? Not critical, we can always revisit this later.